packages/gafl-webapp-service/assets/sass/rod-licensing/_table_extras.scss

```diff
@@ -0,0 +1,7 @@
+.govuk-summary-list__row--last {
+  dd,
+  dt,
+  span {
+    border-bottom: none;
+  }
+}
```

Comment:
Not 100% sure what you're doing here with the styling, but rather than applying a class to the last element, you could use the `:last-child` pseudo element selector and then you don't need to add a class. Have a look at what I've done here: https://codepen.io/jaucourt/pen/xxrJKEN